core/dbt/include/global_project/macros/materializations/models/table/columns_spec_ddl.sql

```diff
@@ -0,0 +1,22 @@
+{%- macro get_columns_spec_ddl() -%}
+    {{ adapter.dispatch('get_columns_spec_ddl', 'dbt')() }}
```

Comment:
This should be a consistent indent (2 spaces).

Reply:
done